bindings/r/LICENSE

```diff
@@ -0,0 +1,2 @@
+YEAR: 2024
+COPYRIGHT HOLDER: treesitterr authors
```

Comment:
Not sure if this is ever destined for CRAN, but I recall BDR getting very huffy about the copyright statement not including all the author names.